internal/provider/virtualmachine/resource_virtualmachine_constructor.go

```diff
@@ -122,6 +122,21 @@ func (c *Constructor) Setup() util.Processors {
 				return nil
 			},
 		},
+		{
+			Field: constants.FieldVirtualMachineReservedMemory,
+			Parser: func(i interface{}) error {
+				reservedMemory := i.(string)
+				if reservedMemory != "" {
+					vmBuilder.Annotations(map[string]string{
+						harvesterutil.AnnotationReservedMemory: i.(string),
```

Comment:
harvesterutil.AnnotationReservedMemory: reservedMemory,

Reply:
Thanks for the check, update according to the suggestion.